Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -743,6 +743,73 @@ public void Editor_InputAsset_CanReplaceBindingGroupThroughSerialization()
         Assert.That(map.bindings[0].groups, Is.EqualTo(""));
     }
 
+    struct AFTestConstants
+    {
+        public const string kOriginalAssetName = "zzMyInputActions";
+        public const string kOriginalDirectory = "zzStartingDirectory";
+
+        public const string kNewAssetName = "NEWactions";
+        public const string kNewDirectory = "NewDirectory";
+
+        public const string kOriginalAssetPath = "Assets/" + kOriginalDirectory + "/" + kOriginalAssetName + "." + InputActionAsset.Extension;
+        public const string kOriginalAssetContents = "{\"name\": \"" + kOriginalAssetName + "\",\"maps\": [],\"controlSchemes\": []}";
+    }
+
+    [Test]
```

Comment:
Great you added these test cases and permutations of tests to cover affected functionality.